plugins/crowdin/src/main.tsx

```diff
@@ -0,0 +1,28 @@
+import "framer-plugin/framer.css"
+
+import { framer } from "framer-plugin"
+import React from "react"
+import ReactDOM from "react-dom/client"
+import { App } from "./App.tsx"
+
+const root = document.getElementById("root")
+if (!root) throw new Error("Root element not found")
+
+const activeLocale = await framer.getActiveLocale()
+const locales = await framer.getLocales()
+
+const isAllowedToSetLocalizationData = framer.isAllowedTo("setLocalizationData")
+
+if (!isAllowedToSetLocalizationData) {
+    framer.notify("You are not allowed to set localization data", { variant: "error" })
+    framer.closePlugin()
+} else if (!activeLocale) {
+    framer.notify("No active locale found", { variant: "error" })
+    framer.closePlugin()
```

Comment:
Combine to avoid two notifications.

<img width="283" height="137" alt="Image" src="https://github.com/user-attachments/assets/90a31ab5-4e6f-4e96-8567-e72428d3b887" />

```suggestion
    framer.closePlugin("No active locale found. Please select a locale.", { variant: "error" })
```